app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,6 @@
 class PostsController < ApplicationController
   def index
-    @user = User.find(params[:user_id])
-    @posts = Post.where(author_id: params[:user_id])
```

Comment:
- [ ] Great job working on this milestone, trying to solve the N+1 problem and adding specs. I think the best approach in solving N+1 problem when fetching all posts and their comments for a user is by using `@posts = @user.posts.includes(:comments)` in the `posts#index`. Also in your index action in the posts_controller, you're not fetching posts but rather user that have made post and comments. Kindly review your approach and if possible take a new screenshot after trying the approach proposed.

Reply:
Done